### PR TITLE
Build scripts for Windows that account for both build types

### LINKED ISSUE #57: Create build scripts that specify the build type for cmake

### PR DESCRIPTION
**Pull request by Dyrand**
Fixes #57 for Windows
I'm not sure this is the way it should be done, but it does work.

### CODE REVIEW

**Comment by MonJamp** [comment]
I forgot we had build scripts. Why not have one script that takes in the build type as an argument? Also what's the point of line 20 in CMakelists.txt?

**Comment by Dyrand** [comment]
The build types defaults to: "Debug;Release;MinSizeRel;RelWithDebInfo"
So I just limited it to the types we work with.
As for a script that takes the build type as an argument, I just didn't think of it.
I'll change it to a script that takes arguments.

File: build.cmd [hunk 1]
```diff
@@ -1,5 +1,7 @@
+set build_type=%1
 mkdir build
 cd build
-cmake ../
-MSBuild revengeMusic.sln
-cd ..
+mkdir %build_type%
```
Comment (MonJamp) [inline build.cmd]: There's no need to separate the CMake generated files. CMake transfers the configurations to VS so you can build both configurations and VS will output them to Debug/ and Release/ receptively. If you change the script up a bit you could run `build Debug & build Release` or via MSBuild `msbuild revengeMusic.sln /p:Configuration=Debug & msbuild revengeMusic.sln /p:Configuration=Release`.

**Review by MonJamp — CHANGES_REQUESTED** [review]

File: build.cmd (hunk 1 above)
Comment (MonJamp) [reply build.cmd]: Also it would be nice to add return to the directory in which the command was run at.

**Review by MonJamp — APPROVED** [review]

**Pull request merged**